static/app/views/issueDetails/traceTimeline/traceIssue.tsx

```diff
@@ -23,21 +24,31 @@ export function TraceIssueEvent({event}: TraceIssueEventProps) {
   const {title, subtitle, message} = getTitleSubtitleMessage(event);
   const avatarSize = parseInt(space(4), 10);
 
-  const referrer = 'issue_details.related_trace_issue';
```

Comment:
maybe we could keep this constant but rename to `issueDetailsReferrer` or something? since it's still used multiple times in this file